package.json

```diff
@@ -86,9 +86,9 @@
     "start-static": "npm run compile:prod && terra express-server --site ./build",
     "test": "npm run jest && npm run wdio",
     "update-package-readme": "node scripts/root-readme-updater/index.js",
-    "wdio-default": "echo wdio default test",
-    "wdio-lowlight": "echo wdio lowlight test",
-    "wdio-fusion": "echo wdio fusion test",
+    "wdio-default": "wdio --spec './packages/terra-clinical-data-grid/tests/wdio/data-grid-spec.js' './packages/terra-clinical-detail-view/tests/wdio/*.js' './packages/terra-clinical-header/tests/wdio/*.js' './packages/terra-clinical-item-display/tests/wdio/*.js'",
+    "wdio-lowlight": "THEME=clinical-lowlight-theme wdio --spec './packages/terra-clinical-data-grid/tests/wdio/data-grid-spec.js' './packages/terra-clinical-header/tests/wdio/*.js' './packages/terra-clinical-item-display/tests/wdio/*.js'",
+    "wdio-fusion": "THEME=orion-fusion-theme wdio --spec './packages/terra-clinical-data-grid/tests/wdio/data-grid-spec.js' './packages/terra-clinical-header/tests/wdio/*.js' './packages/terra-clinical-item-display/tests/wdio/*.js'",
```

Comment:
This will probably have issues scaling with additional PRs. Modifying the `wdio.conf.js` might scale a little better:

```js
// wdio.conf.js

const { config } = require('@cerner/terra-functional-testing/lib/config/wdio.conf');

config.specs = [
 '/packages/terra-clinical-package/tests/wdio/**/*-spec.js*', // Regex or path
];

exports.config = config;
```